### DISCUSSION TITLE
Plural-friendly profile feature

### DISCUSSION THREAD

**Discussion by TheAtomicCat** [crabber-net/crabber#157] [Ideas]
Because of Crabber's prominently large and still growing plural community demographic, i think it'd be a great idea to add plural-friendly features, like "alt profiles" on one account, almost like discord's pluralkit bot, so people can post with a different name/pfp on the same account, possibly while writing the post, they can choose a profile to post it as.

**Comment by hqnna** [comment]
Going to convert this to a discussion, as I believe this needs to be discussed a bit more before any decisions are made.

**Comment by viiuan** [comment]
we've said this offhand on the discord, but i think at the very least a profile switcher should be prioritized over something like this considering how crabber is set up
it works nice on tumblr with sideblogs bc they're all tied to one account but obviously it's one blog per account on here

also just had an idea, but what about instead of completely switching a profile picture or display name, it could kind of function like how submissions on tumblr appear? with the main account's profile shown, but the profile of the member posting shown alongside it. maybe even with appended signatures at the beginning/end of molts or something?
anyway, just ideas lol

**Reply by TheAtomicCat** [reply]
I definitely like this idea. It could also make things look a little neater too I think.
